### DISCUSSION TITLE
The use of US English

### DISCUSSION THREAD

**Discussion by haipinglu** [pykale/pykale#47] [Ideas] [answered]
In our meeting on 28th Jan 2021, we decided to follow the conventions of the PyTorch ecosystem to use **US English**.  For example, two UK-based libraries in the PyTorch ecosystem both use US English:
- See [**MONAI** contribution guideline](https://github.com/Project-MONAI/MONAI/blob/master/CONTRIBUTING.md)  where this is explicitly stated (search "English").
- See *normalize* instead of *normalise* in [**TensorLy** parafac2](https://github.com/tensorly/tensorly/blob/main/tensorly/decomposition/_parafac2.py) (14 appearances).

Therefore, we should try our best to use **US English** from now on and change the previous UK English to US English when convenient. 

We also wonder whether there is any automatic tool for such conversion. We will keep this question open with the [card](https://github.com/pykale/pykale/projects/9#card-53445495) until solved.

@XianyuanLiu You put the following in the card but it is not very clear what it can do. Could you please explain a bit more here in the discussion? Thanks.

PyCharm: Change [Grazie](https://github.com/JetBrains/intellij-community/tree/master/plugins/grazie) setting. (File | Settings | Editor | Proofreading | Grammar | Rules)

**Comment by xianyuanliu** [comment, marked as answer]
PyCharm uses this plugin [Grazie](https://github.com/JetBrains/intellij-community/tree/master/plugins/grazie) to check the spelling. If we would like to use US English in PyCharm, we need to change it's setting. You can find the setting here(File | Settings | Editor | Proofreading | Grammar | Rules). The default setting is both US & UK English.

**Reply by haipinglu** [reply]
Thanks. Yes, we should use US English spell checker from now on and correct if it spotted the need as we proceed.

It may not be necessary to check all files (even so I believe the changes needed are not many, particularly for variable names (maybe more in docstrings).

In VS Code, I use [Code Spell Checker](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker) and its default language setting is US English already: `cSpell.language: en`
